src/web/handlers/FlingGestureHandler.ts

```diff
@@ -18,13 +19,25 @@ export default class FlingGestureHandler extends GestureHandler {
   private maxDurationMs = DEFAULT_MAX_DURATION_MS;
   private minVelocity = DEFAULT_MIN_VELOCITY;
   private minDirectionalAlignment = DEFAULT_MIN_DIRECTION_ALIGNMENT;
+  private enableCornerActivation = DEFAULT_ENABLE_CORNER_ACTIVATION;
```

Comment:
As far as I can see this value is always `true`. Making a prop that sets it doesn't make a lot of sense if you ask me - if I were a user, I'd assume that passing for example `Directions.UP | Directions.LEFT` is enough. 

Is there any situation that we want to disable this behavior? (cc @j-piasecki)

Reply:
Removed all instances of this flag.